scripts/packaging/foundationpreupm.ps1

```diff
@@ -0,0 +1,39 @@
+<#
+.SYNOPSIS
+    Prepares the MRTK foundation for UPM packaging.
+.DESCRIPTION
+    Prepares the MRTK foundation for UPM packaging.
+.PARAMETER PackageRoot
+    The root folder containing the foundation package contents. If not specified, the current folder is presumed.
+#>
+param(
+    [string]$PackageRoot
+)
+
+if (-not $PackageRoot) {
+    throw "Missing required parameter: -PackageRoot."
+}
+
+# Ensure the required folder exists
+$samplesFolder = "$PackageRoot\Samples~"
+if (-not (Test-Path -Path $samplesFolder)) {
+    New-Item $samplesFolder -ItemType Directory | Out-Null
+}
+
+# Copy the scene system's default scenes to Samples~
+Copy-Item -Path "$PackageRoot/SceneSystemResources" -Destination "$samplesFolder/SceneSystemResources" -Recurse -Force
+
+# Create the samples data for the package.json file
+$samples = "`"samples`": ["
+$samples = $samples + "`n      {`n"
+$samples = $samples + "        `"displayName`": `"Scene System Resources`",`n"

```

Comment:
Since all of this is hardcoded, can we just put it directly in the package.json instead of patching it at build-time? Doesn't look like it'll change (as opposed to the examples script that changes dynamically as examples are added).

Reply:
we could

Reply:
fixed by #8780